Examples/Run/Seeding/Common/SeedingExample.cpp

```diff
@@ -109,25 +138,30 @@ int runSeedingExample(int argc, char* argv[],
   seedingCfg.radLengthPerSeed = 0.1;
   seedingCfg.minPt = 500.;
   seedingCfg.bFieldInZ = 0.00199724;
-  seedingCfg.beamPos = {0., 0.};
+  seedingCfg.beamPosX = 0;
+  seedingCfg.beamPosY = 0;
   seedingCfg.impactMax = 3.;
-  seedingCfg.barrelVolume = 8;
-  seedingCfg.barrelLayers = {2, 4, 6};
-  seedingCfg.posEndcapVolume = 9;
-  seedingCfg.posEndcapLayers = {2, 4, 6, 8};
-  seedingCfg.negEndcapVolume = 7;
-  seedingCfg.negEndcapLayers = {14, 12, 10, 8};
-
   sequencer.addAlgorithm(
       std::make_shared<SeedingAlgorithm>(seedingCfg, logLevel));
 
   // Performance Writer
+  TrackFinderPerformanceWriter::Config tfPerfCfg;
+  tfPerfCfg.inputProtoTracks = seedingCfg.outputProtoTracks;
+  tfPerfCfg.inputParticles = particleReader.outputParticles;
+  tfPerfCfg.inputMeasurementParticlesMap =
+      hitSmearingCfg.outputMeasurementParticlesMap;
+  tfPerfCfg.outputDir = outputDir;
+  tfPerfCfg.outputFilename = "performance-trees.root";
```

Comment:
Shall we rename it to e.g. `performance_seeding_trees.root`?

Reply:
Done.